projboard/tests/test_client.py

```diff
@@ -94,6 +94,12 @@ def test_fill_article_delete(client):
     assert '404.html' in template_names
 
 
+@pytest.mark.django_db
+def test_get_my_articles(client):
+    response = client.get("/my_articles/")
```

Comment:
Please consider testing the template of the page itself, and `my-articles` contains the correct number of articles

Reply:
Added template testing at [0ad48e0](https://github.com/redhat-beyond/Smarticle/pull/87/commits/0ad48e0e5f0009f123062371689e7ff61188d84a)
I don't think `my-articles` need additional tests since the articles are already tested in `test_article.py`
WDYT?

Reply:
Sounds great then, I think its useless to test twice

Reply:
but we should consider checking the return result once we have authentication.

Reply:
Please check those returned Values

Reply:
Sorry, currently there is not authentication
Why cant you receive returned errors?

Reply:
Do you mean to test that the length of my_articles equals num_articles?
Like to add: `assert len(set(response.context["my_articles"])) == response.context[COUNT]`?

Reply:
Yes, you return something from the view, isnt it? 
And still you can work with users without authentication
Why is it so important?

Reply:
Thought you meant something else.
This can be (and now has been) done without authentication.

Reply:
You can use this fixture
https://github.com/redhat-beyond/Smarticle/blob/main/projboard/tests/conftest.py#L81

And compare it with the count returned also

Reply:
This fixture counts all the articles in the DB. I test only the articles written by a specific user

Reply:
OK, so why cant you add a user name that you can grab through the get response.

Once you send user name, like 
client.get("/my_articles/test_user")

This case could be tested, but for the search UI,
The page will not be operatable, unless you add the user name

This way, you could have fixture per user you parse as parameter
Count the article and search through UI

Reply:
Please see next comment
Two different tests are better then 2 in one, it is confusing